p4_symbolic/symbolic/packet.h

```diff
@@ -30,7 +32,9 @@ SymbolicPacket ExtractSymbolicPacket(SymbolicPerPacketState state);
 
 // Extract a concrete packet by evaluating every field's corresponding
 // expression in the model.
-ConcretePacket ExtractConcretePacket(SymbolicPacket packet, z3::model model);
+gutil::StatusOr<ConcretePacket> ExtractConcretePacket(
+    SymbolicPacket packet, z3::model model,
+    const values::ValueFormatter &value_formatter);
```

Comment:
Packets should never need translation. After-all, the hardware can't do translation either.